tests/manage/mcg/test_bucket_deletion.py

```diff
@@ -128,22 +139,60 @@ def test_bucket_delete(
             ), f"Found {bucket.name} that should've been removed"
 
     @pytest.mark.parametrize(
-        argnames="interface",
+        argnames="interface, bucketclass_dict",
         argvalues=[
-            pytest.param(*["S3"], marks=[pytest.mark.polarion_id("OCS-1867"), tier3]),
-            pytest.param(*["CLI"], marks=[tier1, pytest.mark.polarion_id("OCS-1917")]),
-            pytest.param(*["OC"], marks=[tier1, pytest.mark.polarion_id("OCS-1868")]),
+            pytest.param(
+                *["S3", None], marks=[tier3, pytest.mark.polarion_id("OCS-1867")]
+            ),
+            pytest.param(
+                *["CLI", None], marks=[tier1, pytest.mark.polarion_id("OCS-1917")]
+            ),
+            pytest.param(
+                *["OC", None], marks=[tier1, pytest.mark.polarion_id("OCS-1868")]
+            ),
+            pytest.param(
+                *[
+                    "OC",
+                    {
+                        "interface": "OC",
+                        "backingstore_dict": {"aws": [(1, "eu-central-1")]},
+                    },
+                ],
+                marks=[tier1],
+            ),
+            pytest.param(
+                *[
+                    "OC",
+                    {"interface": "OC", "backingstore_dict": {"azure": [(1, None)]}},
+                ],
+                marks=[tier1],
+            ),
+            pytest.param(
+                *["OC", {"interface": "OC", "backingstore_dict": {"gcp": [(1, None)]}}],
+                marks=[tier1],
+            ),
         ],
     )
-    def test_bucket_delete_with_objects(self, mcg_obj, interface, awscli_pod):
+    def test_bucket_delete_with_objects(
+        self, mcg_obj, awscli_pod, bucket_class_factory, interface, bucketclass_dict
+    ):
         """
         Negative test with deletion of bucket has objects stored in.
+        ***IMPORTANT***
+        S3 interface is not supporting different bucketclasses DO NOT param S3
```

Comment:
What does it mean? (I don' understand the commend)
Also please add an empty line before and after the comment.

Reply:
This comment is actually misplaced; Comment has been moved and rephrased.